Added the option to view polls (Untested, resubmitted request as bug was found)

Natfan opened the pull request:
Found a bug. Index page doesn't show sharing option. Fixed in this request.

Natfan committed:
Added the option to view closed polls. (Untested)

Natfan committed:
Fixed public index allowing the sharing of polls

ganderzz commented on src/routes/polls.js:
```diff
@@ -42,6 +42,15 @@ module.exports = function(app) {
     })
   );
 
+  app.get(
```
Since this is modifying a poll, thoughts on using a PUT instead of GET ? app.put I believe.

Natfan replied:
Good call!

Natfan committed:
Changed port

Changed port from 3001 to 3000. Used 3001 for testing and forgot to switch it back.

Natfan committed:
Changed /api/polls/:pollId/open from GET to PUT

As recommended.

Natfan committed:
Fixed spelling mistake

`vim` didn't like the emoji and I couldn't get that line right in the terminal. Fixed in browser.

Natfan committed:
Removed newline at end of file.

Natfan reviewed (commented):
Apparently c4b2f41 didn't do anything, oh well...
